WcaOnRails/app/models/competition_event.rb

```diff
@@ -21,4 +21,31 @@ def to_wcif
       "rounds" => self.rounds.map(&:to_wcif),
     }
   end
+
+  def load_wcif!(wcif)
+    # Remove extra rounds.
+    self.rounds.each do |round|
+      wcif_round = wcif["rounds"][round.number - 1]
```

Comment:
In the WCIF spec, round numbers are indexed from 1. However, arrays are indexed from 0, so that's why this `- 1` appears all over the place. Sigh. It's not too late to change this, but I don't know what I'd want to change it to.